docs/CONTRIBUTING.md

```diff
@@ -56,3 +57,40 @@ The following things are to consider before submitting a pull request.
 [black]: https://black.readthedocs.io/en/stable/index.html
 [pep8]: https://www.python.org/dev/peps/pep-0008/
 [mypy]: https://www.mypy-lang.org/
+
+### Commit Message Format
+
+The message format was mainly inspired by the [guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) of Angular.
+
+Please use the following format:
+
+```format
+<type>: <subject>
+```
```

Comment:
One last point: do we specifically want to discourage a commit body beyond the first line?

That's the way I'm reading the format description right now. If we don't we could add a simple `<optional details>` or something like that.

Otherwise, LGTM.

Reply:
like [this](https://github.com/Festo-se/cyclonedx-editor-validator/pull/181/commits/11ca1745302dd91e70e0cd12fb64eea6ac76cb07)?